resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
     END
 
 User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To DBaaSPolicy Screen    ${namespace}
-    Click If Element Present    ${lblOperand}
-    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
-    Click Element    ${btnCreateDBaaSPolicy}
-    Create Custom DBaaSPolicy as ${DBaaSPersona}
+    User Creates DBaaSPolicy For ${DBaaSPersona}
 
 User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Dev View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To Import MongoDB Provider Account On Developer View
     Wait Until Page Contains Element    ${errCreateDBaaSPolicy}          timeout=20s
     Click Element    ${errCreateDBaaSPolicy}
     Create Custom DBaaSPolicy as ${DBaaSPersona}
     User Selects Administrator View
 
+User Creates DBaaSPolicy For ${DBaaSPersona}
+    Click If Element Present    ${lblOperand}
+    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
+    Click Element    ${btnCreateDBaaSPolicy}
+    Create Custom DBaaSPolicy as ${DBaaSPersona}
+
+Max Quota Exceeded Error Displays
+    Scroll Element Into View    ${txtQuotaError}
+    Element Should Be Visible   ${txtQuotaError}
+
+Specify Connection Namespaces And Labels
+    ${ns}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    namespace
+    ${label}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    label
+    IF      ${ns}
+        Click Element    ${btnToggleConnNS}
+        Add Policy Connection Namespaces    ${addNS}
+    END
+    IF    ${label}
+        Click Element    ${btnToggleConnNSSelect}
+        Element Should Be Visible    ${btnToggleMatchExp}
+        Click Element    ${btnToggleMatchExp}
+        Element Should Be Visible    ${btnAddMatchExp}
+        Add Policy Connection Nsselector    ${addLabel}
```

Comment:
Change 'Nsselector' to 'NsSelector'

Reply:
Method "add_policy_connection_nsselector" is a low level keyword and while calling it as middle level we should use Title case. This change was introduced by Robotidy, even If we change it now the same change would be introduced while running robotidy.

Reply:
OK got it.